example-apps/app-react/README.md

```diff
@@ -0,0 +1,59 @@
+# Example project with ReactJS
+
+The goal of this project is to show how to use the Svelte components in a React project.
+
+
+## How to add new component
+
+- Import UI library and adapter function:
+
+```js script
+import '@sprinteins/dpdhl-uilib';
+import {adapt} from "./adapter";
+```
+
+- Initialise a component using an adapter function and a name of the component:
+
+```js script
+const Pagination = adapt('dpdhl-pagination')
```

Comment:
There is no need to adapt it yourself
All components should be already exported as adapted React components
We only need to import from `/react`
```ts
import { Pagination } from '@sprinteins/dpdhl-uilib/react' 
```